src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -67,4 +80,20 @@ public void patchObjection(String requestId, String companyNumber, String object
         }
     }
 
+    @Override
+    public ServiceResult<String> addAttachment(String objectionId, MultipartFile file) throws ServiceException {
+        FileTransferApiClientResponse response = fileTransferApiClient.upload(objectionId, file);
+
+        HttpStatus responseHttpStatus = response.getHttpStatus();
+        if (responseHttpStatus != null && responseHttpStatus.isError()) {
+            throw new ServiceException(responseHttpStatus.toString());
+        }
+        String fileId = response.getFileId();
+        if (StringUtils.isBlank(fileId)) {
+            throw new ServiceException("No file id returned from file upload");
```

Comment:
I noticed in extensions-api attachments controller.upload endpoint, it is catching
catch(HttpClientErrorException | HttpServerErrorException e) 

assuming the code is similar, we would need to handle these exceptions either here or somewhere further up the chain possibly in the fileTransferClient